packages/web-runtime/src/container/sse.ts

```diff
@@ -203,3 +206,88 @@ export const onSSEProcessingFinishedEvent = async ({
     console.error(`Unable to parse sse event ${topic} data`, e)
   }
 }
+
+export const onSSEItemTrashedEvent = ({
+  topic,
+  language,
+  messageStore,
+  resourcesStore,
+  msg
+}: {
+  topic: string
+  language: Language
+  resourcesStore: ResourcesStore
+  messageStore: MessageStore
+  msg: MessageEvent
+}) => {
+  try {
+    const sseData = fileReadyEventSchema.parse(JSON.parse(msg.data))
+    const currentFolder = resourcesStore.currentFolder
+    const resourceIsCurrentFolder = currentFolder.id === sseData.itemid
+
+    if (resourceIsCurrentFolder) {
+      return messageStore.showMessage({
```

Comment:
Found one issue here: When I am in folder `a/b/c` and delete `c` from the breadcrumb menu, I get two messages in the bottom right:
- success message that the folder was deleted (+ a navigation to `a/b` happens automatically because I triggered the delete action myself)
- this warning here that my current location was removed (although I was already navigated up)

Any idea how to solve this?

Reply:
Ugh no other idea then adding a delay :(

Reply:
then let's try with a slight delay on the SSE handling.

Reply:
Sure. Later on we should work with client ids and early return if the original event was initiated by the same client id

Reply:
Yep, nice idea. Do you have an ocis ticket for that already? if not, please create one. 👍

Reply:
Just popped up in my mind 5 minutes ago lol

Reply:
The issue will be created tomorrow, work around has been implemented. Please re-review